src/test/regress/sql/multi_insert_select_conflict.sql

```diff
@@ -1,3 +1,11 @@
+--
+-- MULTI_INSERT_SELECT_CONFLICT
+--
+-- This test file has an alternative output because of the change in the
```

Comment:
In addition to this comment you should include a version check at the top, like we did in the past here:
https://github.com/citusdata/citus/commit/f9b02946b125483e1439d374129541c893592db4#diff-bddd1d49aa3304cb7531b4672f2ef9a6a502a8f75f03e1b03e6e4cfa4e191ee3L5-L8

Otherwise when an error occurs due to an actual bug it might compare against the wrong output file and then the diff will be much larger than intended.